CONTRIBUTING.md

```diff
@@ -2,10 +2,41 @@
 
 Thanks for being willing to contribute to [Clerk's documentation](https://clerk.com/docs)! This document outlines how to effectively contribute updates and fixes to the documentation content located in this repository.
 
+Table of Contents
```

Comment:
nit: GitHub renders a table of contents in its UI (the button on the top right toggles it)

<img width="1173" alt="Screenshot 2023-08-21 at 2 09 26 PM" src="https://github.com/clerkinc/clerk-docs/assets/1289701/a30e471b-71db-414f-8d26-20b99a7f0d3b">